src/org-manager/services/pba.service.ts

```diff
@@ -17,7 +17,7 @@ export class PbaService {
   }
 
   public setPBAStatus(body: any): Observable<any> {
-    return this.http.put<any>(`${this.pbaStatusUrl}`, body);
+     return this.http.put(`${this.pbaStatusUrl}`, body);
```

Comment:
rather than remove the `any` type isn't there a way you could strongly type the response?

Reply:
Added a ConfirmNewPBAModel Model and strongly typed it in the response.